sponsor_us.md

```diff
@@ -17,40 +17,46 @@ By supporting us you help to make the world a little better. Your company also
 gets to associate itself with a positive and ambitious event, which aims to
 increase diversity in the technology scene.
 
-Sponsors are listed on our website and on our event page on ClojureBridge.org.
-They receive a shoutout at the start and end of the event, and receive a warm
-"thank you" on our social media account. We also list sponsors by name on our
-popular t-shirts that are handed out to teachers and attendees, and provide the
-opportunity to offer "swag" (stickers, note books, thumb drives, etc.) at the
-event.
+Your company has an option of different tiers of support. There's the direct
+sponsorship tiers, which are the most valuable form of support, and a generic
+small supporter tier. The direct sponsorship tiers involve directly paying our
+supplier with the order details we provide, and as such also mean you can pay
+with credit card will and receive an invoice.
```

Comment:
typo: and will receive

Also... that's not quite correct. They will receive an invoice either way. Maybe "an invoice from them directly" (aka the supplier).